source/components/molecules/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import styled, { ThemeContext } from 'styled-components';
+import { View, TouchableHighlight } from 'react-native';
+import RadioButton from '../../atoms/RadioButton/RadioButton';
+import Text from '../../atoms/Text/Text';
+import { getValidColorSchema } from '../../../styles/theme';
+
+const TouchableWrapper = styled(TouchableHighlight)`
+  border-radius: 4px;
+  padding: 2px;
+`;
+const Row = styled(View)`
+  flex-direction: row;
+  align-items: center;
+  margin-bottom: 3px;
+  margin-top: 3px;
+`;
+const ButtonWrapper = styled(View)`
+  flex: 1;
+`;
+const TextWrapper = styled(View)<{ size: 'small' | 'medium' | 'large' }>`
+  flex: 10;
+  margin-left: ${props => props.theme.radiobuttonGroup[props.size].textMargin}px;
+`;
+
+interface Props {
+  choices: { displayText: string; value: string }[];
+  value?: string;
+  onSelect: (value: string) => void;
+  colorSchema?: string;
+  size?: 'small' | 'medium' | 'large';
+}
+
+/**
+ * A component for displaying a number of choices, where at most one can be selected at a time.
+ */
+const RadioGroup: React.FC<Props> = ({ choices, value, onSelect, colorSchema, size }) => {
+  const validColorSchema = getValidColorSchema(colorSchema);
+  const theme = useContext(ThemeContext);
+  return (
+    <>
+      {choices.map(choice => (
+        <TouchableWrapper
+          key={choice.value}
+          onPress={() => onSelect(choice.value)}
+          underlayColor={theme.colors.complementary[validColorSchema][1]}
+          activeOpacity={0.6}
+        >
+          <Row>
+            <ButtonWrapper>
+              <RadioButton
+                onSelect={() => onSelect(choice.value)}
+                selected={value === choice.value}
+                colorSchema={validColorSchema}
+                size={size || 'small'}
+              />
+            </ButtonWrapper>
+            <TextWrapper size={size || 'small'}>
+              <Text type={theme.radiobuttonGroup[size || 'small'].textType}>
+                {choice.displayText}
+              </Text>
+            </TextWrapper>
+          </Row>
+        </TouchableWrapper>
+      ))}
+    </>
+  );
+};
+RadioGroup.propTypes = {
+  /** The array of choices, each entry corresponding to one row */
+  choices: PropTypes.arrayOf(
+    PropTypes.shape({
+      displayText: PropTypes.string,
+      /** The value to store in the context/backend */
+      value: PropTypes.string,
+    })
+  ).isRequired,
+  value: PropTypes.string,
+  onSelect: PropTypes.func,
+  colorSchema: PropTypes.string,
+  size: PropTypes.oneOf(['small', 'medium', 'large']),
```

Comment:
Sorry but i disagree with @dannil76 in this matter, the reason we want comments on propTypes is becouse the naming can be very concise and therefore hard to understand at a first glance. As a bonus the propTypes comments can be used with JSDoc to generate component documentation regarding props. Most of the component libraries out there follow these simple rules, since the propTypes are the API for an developer to use the Component and therefore it must be clear how to use it :)

Reply:
Again we are missing documentation and conventions. So i understand the request of change since we look at this from different views. It's not my decision to take if we should or should not have propTypes comment i just raise the reason why i believe it's worth having (Y)

Reply:
So to be clear my standpoint is that propTypes documentation is important for understanding the usage of a component.

Reply:
> Sorry but i disagree with @dannil76 in this matter, the reason we want comments on propTypes is becouse the naming can be very concise and therefore hard to understand at a first glance. As a bonus the propTypes comments can be used with JSDoc to generate component documentation regarding props. Most of the component libraries out there follow these simple rules, since the propTypes are the API for an developer to use the Component and therefore it must be clear how to use it :)

Sorry for my ignorance. Then this is not a problem.

Reply:
Sorry, I'm getting confused here... I sympathise with both viewpoints, in that I feel too obvious comments are just superfluous noise. I initially added these comments because I knew what Teddy thinks about them, but at the same time, if a prop has the name displayText, then a comment telling you that it's the text that gets displayed just feels very redundant, both in the file but also in a JSDoc... Similarly for value, size and so on: the comment is just restating what is already in the name, which makes it just noise. 

But okay, to move forward, let's follow the standard that all props should have some documentation. I'll add back some small comments, and then I'll merge this PR. All in all, this issue is a bit too small to get hang up on, I think.